src/main/java/org/spoken_tutorial/health/elasticsearch/models/QueueManagement.java

```diff
@@ -0,0 +1,236 @@
+package org.spoken_tutorial.health.elasticsearch.models;
+
+import java.sql.Timestamp;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+
+@Entity
+public class QueueManagement {
+
+    @Id
+    @Column(name = "queueId", nullable = false, updatable = false)
+    private long queueId;
+
+    @Column(name = "requestTime", nullable = true, updatable = false)
+    private Timestamp requestTime;
+
+    @Column(name = "requestType", nullable = true)
+    private String requestType;
+
+    @Column(name = "status", nullable = true)
+    private String status;
+
+    @Column(name = "startTime", nullable = true)
+    private long startTime;
+
+    @Column(name = "endTime", nullable = true)
+    private long endTime;
+
+    @Column(name = "procesingTime", nullable = true)
+    private long procesingTime;
+
+    @Column(name = "message", nullable = true)
+    private String message;
+
+    @Column(name = "documentId", nullable = true)
+    private String documentId;
+
+    @Column(name = "documentType", nullable = true)
+    private String documentType;
+
+    @Column(name = "documentPath", nullable = true)
+    private String documentPath;
+
+    @Column(name = "documentUrl", nullable = true)
+    private String documentUrl;
+
+    @Column(name = "rankView", nullable = true)
+    private int rank;
+
+    @Column(name = "view_url", nullable = true)
+    private String view_url;
```

Comment:
```suggestion
    @Column(name = "viewUrl", nullable = true)
    private String viewUrl;
```
rename this to viewUrl